sql_runner/snowflake_target.go

```diff
@@ -105,15 +105,12 @@ func (sft SnowflakeTarget) RunQuery(query ReadyQuery, dryRun bool, showQueryOutp
 	}
 
 	// Enable grabbing the queryID
-	queryIDChannel := make(chan string)
+	queryIDChannel := make(chan string, 1)
```

Comment:
I presume this is to enforce "there should only be one queryID per script"?

Reply:
Yes i think so. The pattern here is copied directly from how [it is being tested](https://github.com/snowflakedb/gosnowflake/blob/d62ff8279fcb2f72c7bd1b0c85354eb64e4e5f88/statement_test.go#L60) in gosnowflake. Also thought it may make sense to use a buffered channel to not block in case the value is never received?